src/platform/mock-local-home-platform.ts

```diff
@@ -86,4 +94,63 @@ export class MockLocalHomePlatform {
     this.localDeviceIds.set(device.id, device.verificationId);
     return device.verificationId;
   }
+
+  /**
+   * Forms an `ExecuteRequest` with the given parameters and passes it to the fulfillment app
+   * @param requestId  The request id to set in the `ExecuteRequest`
+   * @param deviceId  The device id to set in the `ExecuteRequest`
+   * @param command  The command string to set as the single command in the `ExecuteRequest`
+   * @param params  The execution parameters to set in the single command in the `ExecuteRequest`
+   */
+  public async triggerExecute(
+    requestId: string,
+    deviceId: string,
+    command: string,
+    params: object
+  ): Promise<smarthome.IntentFlow.ExecuteStatus> {
+    // Cannot send a `ExecuteRequest` to a device not registered
+    if (!this.localDeviceIds.has(deviceId)) {
+      throw new Error(ERROR_DEVICE_ID_NOT_REGISTERED);
+    }
+
+    // No executeHandler found
+    if (this.app.executeHandler === undefined) {
+      throw new Error(ERROR_UNDEFINED_EXECUTE_HANDLER);
+    }
+
+    const executeRequest: smarthome.IntentFlow.ExecuteRequest = {
+      requestId: requestId,
+      inputs: [
+        {
+          intent: smarthome.Intents.EXECUTE,
+          payload: {
+            commands: [
+              {
+                execution: [
+                  {
+                    command,
+                    params,
+                  },
+                ],
+                devices: [
+                  {
+                    id: deviceId,
+                  },
+                ],
+              },
+            ],
+            structureData: {},
```

Comment:
I believe so.  It's `@hidden` and omitted from all documentation.  I'm assuming it's used internally.

Reply:
If it's not in the documentation and is hidden, should we be sending it?

Reply:
It's required in order to form an `IdentifyRequest`, since the `Payload<LocalIdentifiedDevice>` type requires it.  I could stub `IdentifyRequest` if we want to remove it.